Cargo.toml

```
@@ -38,6 +38,7 @@ rand = "0.8"
 rumqttc = { version = "0.17", default-features = false }
 rustls = { version = "0.20", features = ["dangerous_configuration"], optional = true }
 rustls-native-certs = { version = "0.6", optional = true }
+rustls-pemfile = { version = "1.0.1", optional = true }
```

Personally I started using the major component of semver in the cargo.toml and let the lock file handle the specifics. dependency updates only require changing the lock file then

```suggestion
rustls-pemfile = { version = "1", optional = true }
```